### PR TITLE
Renames deprecated $call to $callable in nock

### PR DESCRIPTION
**Pull request by arekkas**

### CODE REVIEW

**Comment by AndrewSouthpaw** [comment]
There's lots of test errors, those will need to be fixed before this can get merged in.

**Comment by arekkas** [comment]
Yeah I just saw that, absolutely no idea what or why or how it's failing. But what I noticed is that you're running flow `0.62.0`? I was testing this with `^0.75.0` locally which seemed to work fine.

File: definitions/npm/nock_v9.x.x/flow_v0.31.x-/nock_v9.x.x.js [hunk 1]
```diff
@@ -38,13 +38,13 @@ declare type $npm$nock$InterceptorOptions = {
 };
 
 declare class $npm$nock$NockBack {
-  $call(path: string, cb: (cb: Function) => any): void,
+  $callable(path: string, cb: (cb: Function) => any): void,
```
Comment (AndrewSouthpaw) [inline definitions/npm/nock_v9.x.x/flow_v0.31.x-/nock_v9.x.x.js]: You'll need to cap this at `flow_v0.74`, and then copy this file over to a new folder `flow_v0.75.x-`. I think `$callable` was introduced in `0.75` yeah?

Comment (arekkas) [reply definitions/npm/nock_v9.x.x/flow_v0.31.x-/nock_v9.x.x.js]: Ah ok, that makes sense! I'll try it out.

**Comment by AndrewSouthpaw** [comment]
You can run the library tests with `./run_def_tests [library_name]` to help you with debugging.

**Comment by arekkas** [comment]
Wasn't able to get it working on my windows machine, so I'll just squash & force-push the hell out of this one and let you know once it passes.

**Comment by arekkas** [comment]
Tests now fail with a timeout (`Timeout - Async callback was not invoked within the 10000ms timeout specified by jest.setTimeout.`) which appears to be related to the cache? Not sure...I'm out of office until monday but will take another look after that. Cheers!

**Comment by GAntoine** [comment]
Closing as these changes have already been made in another PR.

**Pull request closed without merging**